src/main/java/tutorlink/commandpackage/AddCourseCommand.java

```diff
@@ -30,7 +31,7 @@ public CommandResult execute() {
             Course course = new Course(this.courseID, this.courseMCs);
             CourseList courseList = students.getStudent(this.matricNumber).courses;
             courseList.addCourse(course);
-            return new CommandResult(String.format(SUCCESSFUL_MESSAGE,course));
+            return new CommandResult(String.format(SUCCESSFUL_MESSAGE,course,this.matricNumber));
```

Comment:
I like that you incorporated the matricNumber in the print string as it allows us to display more information